magento/magento2
# 11793: Magento2.1.5 admin shipping report shows wrong currency code

Admin reports show only default currency code.

### Fixed Issues (if relevant)
1. magento/magento2#11793: Magento2.1.5 admin shipping report shows wrong currency code

### Manual testing scenarios
Steps to reproduce:
1. Create a single website with multi-store views (2 stores).
2. Go to Stores-Configuration->General->Currency Setup->Currency Options and set it up to show USD currency for the 1 store, and EUR currency (as an example) for the second store.
3. Create 2 orders for each store (go to Sales->Orders), invoices and shippings for them.
4. Go to Admin->Reports->Sales->Shipping and set up the filters so you can see shipments for that orders you created earlier.
5. Currency must change depends on store.

### Contribution checklist
